.travis.yml

```diff
@@ -25,7 +26,18 @@ before_install:
   # Changing the value for sudo will also need to be reviewed at the same time
   # This will need to be reviewed when the issue above is fixed
   - export PATH=/usr/bin/:$PATH
-  - sudo apt-get install -y zeroc-ice35
+  - sudo apt-get -y install python-{pillow,numpy,psutil}
+  - sudo apt-get -y install python-dev build-essential
+  - sudo apt-get -y install db5.3-util
+  - sudo apt-get -y install libssl-dev libbz2-dev libmcpp-dev libdb++-dev libdb-dev
+
+  - sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5E6DA83306132997
+  - sudo apt-add-repository "deb http://zeroc.com/download/apt/ubuntu`lsb_release -rs` stable main"
+  - sudo apt-get update
+  - sudo apt-get -y install zeroc-ice-all-runtime zeroc-ice-all-dev
+  #- sudo pip install -U "zeroc-ice<3.7"
+  - sudo curl -L -O https://gitlab.com/openmicroscopy/incubator/zeroc-ice-py-ubuntu1604/-/jobs/129805187/artifacts/raw/dist/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
+  - sudo pip install -U zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
```

Comment:
FYI you can pip install a URL directly `pip install -U https://gitlab.com/openmicroscopy/incubator/zeroc-ice-py-ubuntu1604/-/jobs/129805187/artifacts/raw/dist/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl`

Reply:
I used two lines mainly for clarity at that stage.
That will be replaced when we have decided of the final location

Reply:
I did a bit of investigation:
* 5.2 server will not start on Xenial travis docker image due to the Java version installed by default.
* An option is to keep the 14.04 image and use the wheel for 14.04 (we will need for https://github.com/openmicroscopy/openmicroscopy/)
* then move to 16.04 image when we have 5.5 out so we can do an upgrade from 5.3->5.5

What do you think?